docs/widgets/baseNodes/element.md

```diff
@@ -0,0 +1,201 @@
+#### Element (from Node)
```

Comment:
Add hyperlink on the parent classes (they can be put on the title itself without problems)